terraform/mesh_aws/locals.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  name = "${var.name_prefix}-mesh"
+
+  mesh_url = {
+    integration = "https://msg.intspineservices.nhs.uk"
+    production  = "https://msg.spineservices.nhs.uk"
```

Comment:
should be
`https://mesh-sync.spineservices.nhs.uk`

Reply:
https://github.com/NHSDigital/spine-core-aws-common/pull/22#discussion_r665508653

Reply:
@ramensen  is wrong ;) I am right :)

Reply:
Sorry, I said it from memory - always a bad move!